craft_parts/actions.py

```diff
@@ -63,9 +66,12 @@ class Action:
     :param action_type: Action to run for this step.
     :param reason: A textual description of why this action should be
         executed.
+    :param project_vars: The values of project variables from a previous
+        execution if the action type is ``ActionType.SKIP``.
```

Comment:
this makes it seem like it is a failure to set this if the action type is not SKIP, we either should have a guard for this or edit the text (useful for SKIP, best used with, or no comment at all).

Reply:
Changed to "... from a previous execution, used if action type is..."